src/back/msl/writer.rs

```diff
@@ -1339,7 +1338,10 @@ impl<W: Write> Writer<W> {
                 let separator = separate(last_used_global == Some(handle));
                 write!(self.out, "{}", INDENT)?;
                 tyvar.try_fmt(&mut self.out)?;
-                resolved.try_fmt_decorated(&mut self.out, separator)?;
+                if var.binding.is_some() {
```

Comment:
Is this change intentionally included in this PR?

Reply:
not really, I'll move it out